src/Magnum/Audio/Source.h

```diff
@@ -734,6 +758,12 @@ inline Source& Source::operator=(Source&& other) {
     return *this;
 }
 
+inline auto Source::type() const -> Type {
```

Comment:
As we discussed on Gitter, can you add a simple test for this? Thanks!

Reply:
I forgot to push it. 🤦‍♂